mods/cnc/tilesets/winter.yaml

```diff
@@ -449,8 +449,8 @@ Templates:
 		Size: 2,2
 		Category: Cliffs
 		Tiles:
-			0: Clear
-			1: Clear
+			0: Rock
```

Comment:
Here on this three tiles again:
![tdcliff5](https://cloud.githubusercontent.com/assets/7704140/12011138/19a3181a-acc3-11e5-98dc-b4262ac62558.png)